scripts/showcase_config.json

```diff
@@ -2,30 +2,46 @@
     "proj-refs": {
         "Example/Example/Example.csproj": [
             "..\\..\\Core\\Core\\Core.csproj",
-            "..\\..\\Auth\\Auth\\Auth.csproj"
+            "..\\..\\Auth\\Auth\\Auth.csproj",
+            "..\\..\\Security\\Security\\Security.csproj"
         ],
         "Example/Example.Android/Example.Android.csproj": [
             "..\\..\\Core\\Core.Platform.Android\\Core.Platform.Android.csproj",
             "..\\..\\Core\\Core\\Core.csproj",
             "..\\..\\Auth\\Auth\\Auth.csproj",
+            "..\\..\\Security\\Security\\Security.csproj",
+            "..\\..\\Security\\Security.Platform.Android\\Security.Platform.Android.csproj",
             "..\\..\\Auth\\Auth.Platform.Android\\Auth.Platform.Android.csproj"
         ],
         "Example/Example.iOS/Example.iOS.csproj": [
             "..\\..\\Core\\Core.Platform.iOS\\Core.Platform.iOS.csproj",
             "..\\..\\Core\\Core\\Core.csproj",
             "..\\..\\Auth\\Auth\\Auth.csproj",
+            "..\\..\\Security\\Security\\Security.csproj",
+            "..\\..\\Security\\Security.Platform.iOS\\Security.Platform.iOS.csproj",
             "..\\..\\Auth\\Auth.Platform.iOS\\Auth.Platform.iOS.csproj"]
     },
     "nuget-deps": {
         "Example/Example/Example.csproj": {
-            "AeroGear.Mobile.Core": "0.0.6",
-            "AeroGear.Mobile.Auth": "0.0.6"
+            "AeroGear.Mobile.Core": "1.0.0-alpha",
+            "AeroGear.Mobile.Auth": "1.0.0-alpha",
+            "AeroGear.Mobile.Security": "1.0.0-alpha"
         },
         "Example/Example.Android/Example.Android.csproj": {
-            "AeroGear.Mobile.Auth.Platform.Android": "0.0.6"
+            "AeroGear.Mobile.Core": "1.0.0-alpha",
+            "AeroGear.Mobile.Core.Platform.Android": "1.0.0-alpha",
+            "AeroGear.Mobile.Auth": "1.0.0-alpha",
+            "AeroGear.Mobile.Auth.Platform.Android": "1.0.0-alpha",
+            "AeroGear.Mobile.Security": "1.0.0-alpha",
+            "AeroGear.Mobile.Security.Platform.Android": "1.0.0-alpha"
         },
         "Example/Example.iOS/Example.iOS.csproj": {
-            "AeroGear.Mobile.Auth.Platform.iOS": "0.0.6"
+            "AeroGear.Mobile.Core": "1.0.0-alpha",
+            "AeroGear.Mobile.Core.Platform.iOS": "1.0.0-alpha",
+            "AeroGear.Mobile.Auth": "1.0.0-alpha",
+            "AeroGear.Mobile.Auth.Platform.iOS": "1.0.0-alpha",
+            "AeroGear.Mobile.Security": "1.0.0-alpha",
+            "AeroGear.Mobile.Security.Platform.iOS": "1.0.0-alpha"                                                    
```

Comment:
should this script be part of the showcase app repo?

Reply:
I had it there before. But I decided that this is better place. Script is used only for the development. Also it shares some code with the SDK release script.